src/Symfony/Component/Serializer/Mapping/AttributeMetadata.php

```diff
@@ -127,17 +127,71 @@ public function getMaxDepth(): ?int
     /**
      * {@inheritdoc}
      */
-    public function setSerializedName(string $serializedName = null)
+    public function setSerializedNames(array $serializedNames): void
     {
-        $this->serializedName = $serializedName;
+        $this->serializedNames = $serializedNames;
+    }
+
+    /**
+     * Set a serialization name for given groups.
+     *
+     * @param string[] $groups
+     */
+    public function setSerializedName(string $serializedName = null/* , array $groups = [] */)
+    {
+        if (\func_num_args() < 2) {
+            trigger_deprecation('symfony/serializer', '6.2', 'The "%s()" method will have a new "array $groups = []" argument in version 7.0, not defining it is deprecated.', __METHOD__);
```

Comment:
It should be legit to call this method with only one argument, isn't it?
then this deprecation should be removed. DebugClassLoader will trigger one anyway when needed